browser/components/GroupsTable.tsx

```diff
@@ -87,11 +133,24 @@ class GroupsTable extends React.Component<Props> {
               Header: <b># Contribs YTD</b>,
               accessor: 'contribYear',
             },
+            {
+              Header: <b>Report</b>,
+              accessor: 'group_id',
+              width: 80,
+              Cell: row => (
+                <div className="center">
+                  <Link to="/strategicprojects" onClick={this.handleDownload}>
```

Comment:
the link should be "strategic/projects".

Reply:
The api link is strategic/projects but I had the actual website link go to just strategicprojects. Do you want me to change the actual frontend url as well?

Reply:
Since I can't come up with a reason for why we should do 'strategic/projects', I am going to instead request that you just add a hyphen between. So it will be 'strategic-projects'.

Reply:
Sounds good!